entry_test.go

```diff
@@ -267,7 +267,7 @@ var _ = Describe("Tests", func() {
 			It("should locate", func() {
 				location, err := integer.GetLocation()
 				Expect(err).ToNot(HaveOccurred())
-				Expect(int16(qdbPort)).To(Equal(location.Port))
+				Expect(int16(2836)).To(Equal(location.Port))
```

Comment:
So here you suppose insecure cluster. Why can't you make it a parameter/variable?

Reply:
I don't like the idea of creating a global variable for only one place in the code.

Reply:
Do you prefer magic numbers then?